wcivf/apps/elections/templates/elections/includes/_voter_id.html

```diff
@@ -25,6 +25,21 @@ <h2 id="requirements">
                     </li>
                 </ul>
             {% endblocktrans %}
+
+            {% if not postelections.0.past_vac_application_deadline %}
```

Comment:
Here I am using the property that all elections that use ID (usefully) have the same VAC deadline